htdocs/core/class/commonobject.class.php

```diff
@@ -7829,14 +7829,15 @@ function handlemultiinputdisabling(htmlname){
 						$sqlwhere .= " AND entity = " . ((int) $conf->entity);
 					}
 					$sql .= $sqlwhere;
-					//print $sql;
 
 					// Note: $InfoFieldList can be 'sellist:TableName:LabelFieldName[:KeyFieldName[:KeyFieldParent[:Filter[:CategoryIdType[:CategoryIdList[:Sortfield]]]]]]'
 					if (isset($InfoFieldList[7]) && preg_match('/^[a-z0-9_\-,]+$/i', $InfoFieldList[7])) {
 						$sql .= " ORDER BY ".$this->db->escape($InfoFieldList[7]);
 					} else {
 						$sql .= " ORDER BY ".$this->db->sanitize(implode(', ', $fields_label));
 					}
+					$sql .= ' LIMIT ' . getDolGlobalString('MAIN_EXTRAFIELDS_LIMIT_SELLIST_SQL', 1000);
```

Comment:
getDolGlobalInt instead of getDolGlobalString to avoid sql injection
or
(int) getDolGlobalString('MAIN_EXTRAFIELDS_LIMIT_SELLIST_SQL', 1000);